saagieapi/pipelines/pipelines.py

```diff
@@ -32,8 +33,9 @@ def list_for_project(self, project_id: str, instances_limit: int = -1) -> Dict:
         params = {"projectId": project_id}
         if instances_limit != -1:
             params["instancesLimit"] = instances_limit
-        query = gql(GQL_LIST_PIPELINES_FOR_PROJECT)
-        return self.saagie_api.client.execute(query, variable_values=params)
+        return self.saagie_api.client.execute(
+            query=gql(GQL_LIST_PIPELINES_FOR_PROJECT), variable_values=params, pprint_result=True
```

Comment:
`pprint_result` as parameter ?

Reply:
not needed as this function is not called by other functions